site/docs/guides/customize-materialization-fields.md

```diff
@@ -0,0 +1,135 @@
+# Customize materialized fields
+
+When you first materialize a collection to an endpoint like a database or data warehouse,
+the resulting table columns might not be formatted how you want.
+You might notice missing columns, extra columns, or columns with names you don't like.
+This happens when the default collection schema doesn't map to a table schema appropriate for your use case.
+
+You can control the shape and appearance of materialized tables using a two-step process.
```

Comment:
I'm not sure exactly where this should go, but I feel like there should be something to the effect of "if you just want to add a field that isn't being included by default, then skip ahead to 'include desired fields in your materialization'(linky)"